uktrade/data-hub-frontend
# Replace typeaheads - swap out old typeaheads for the new one

## Description of change

Uses the new accessible typeahead instead of the old ones. 

Note that the old typeahead is still in place in a few places around datahub - these can be replaced in a later ticket once we know people are happy with these on the collection list pages.

This also required a few changes ot the new typeahead to deal with RoutedTypeaheads / onChange events etc.

## Test instructions

Go to the various collection list pages and try out the typeaheads. These should now be using the accessible typeaheads.

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/1234577/148095892-9d16daad-6789-4f16-8fba-e9d089501986.png)

### After

![image](https://user-images.githubusercontent.com/1234577/148096023-bd6c74e2-abce-4af2-8838-aed060869698.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
